numpy/lib/utils.py

```diff
@@ -13,9 +13,86 @@
 __all__ = [
     'issubclass_', 'issubsctype', 'issubdtype', 'deprecate',
     'deprecate_with_doc', 'get_include', 'info', 'source', 'who',
-    'lookfor', 'byte_bounds', 'safe_eval'
+    'lookfor', 'byte_bounds', 'safe_eval', 'show_runtime'
     ]
 
+
+def show_runtime():
+    """
+    Print information about various resources in the system
+    including available intrinsic support and BLAS/LAPACK library
+    in use
+    See Also
+    --------
+    show_config : Show libraries in the system on which
+                  NumPy was built.
+    Notes
```

Comment:
Blank line, reformat
```suggestion
    show_config : Show libraries in the system on which NumPy was built.

    Notes
```

Reply:
Added in [fe63699](https://github.com/numpy/numpy/pull/21468/commits/fe6369961f4ff136161b0dac2ae2ec620dbce5fb)